app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -39,6 +39,14 @@ def show
         course_option_id: @application_choice.offered_option.id,
       )
 
+      if @application_choice.status == 'offer_deferred'
+        @deferred_offer_wizard_applicable =
+          @application_choice.recruitment_cycle == RecruitmentCycle.previous_year
```

Comment:
Maybe a `can_be_reinstated?` method?

Reply:
After some thinking, it feels a bit niche to attach this to `ApplicationChoice`.